OasisDEX/oasis-borrow
# Rewrite portfolio to use debank API for Assets Overview section

## Changes 👷‍♀️

- Added new api endpoint service to provide wallet and protocol assets overview data. This data is using API secret auth key and also provide simple caching to optimize Debank API usage costs which are per user call.
- Added new API key for debank API
- Refactored Assets Overview Page to use and display new data from Debank instead of legacy rxjs pipes
- Cleand up all legacy rxjs pipes responsible for preparing data for assets overview
  
## How to test 🧪

- open portfolio page and check the overview section is loadin correctly
- debank api calls should be cached for 60 sec on api level and in the browser using cache control header

PS: ask me for debank key
